Framework/ScriptRepository/inc/MantidScriptRepository/ScriptRepositoryImpl.h

```diff
@@ -106,7 +106,7 @@ class SCRIPT_DLL_EXPORT ScriptRepositoryImpl : public ScriptRepository {
 
   void setIgnorePatterns(const std::string &patterns) override;
 
-  std::string ignorePatterns() override;
+  std::string ignorePatterns() override final;
```

Comment:
```suggestion
  std::string ignorePatterns() final;
```
`final` enforces `override`, so you don't need both.